src/uu/dmesg/src/dmesg.rs

```diff
@@ -196,6 +201,7 @@ impl Dmesg<'_> {
     fn new() -> Self {
         Dmesg {
             kmsg_file: "/dev/kmsg",
+            kmsg_record_separator: 10, // '\n'
```

Comment:
You can write it more clearly like so:

```suggestion
            kmsg_record_separator: b'\n',
```